freeplane/src/main/java/org/freeplane/view/swing/map/overview/ResizableBorder.java

```diff
@@ -0,0 +1,146 @@
+package org.freeplane.view.swing.map.overview;
+
+import java.awt.Color;
+import java.awt.Component;
+import java.awt.Cursor;
+import java.awt.Dimension;
+import java.awt.Graphics;
+import java.awt.Insets;
+import java.awt.Point;
+import java.awt.Rectangle;
+import java.awt.event.MouseEvent;
+
+import javax.swing.SwingConstants;
+import javax.swing.border.Border;
+
+import org.freeplane.view.swing.map.MapView;
+
+public class ResizableBorder implements Border, SwingConstants {
+    static final int SIZE = 6;
```

Comment:
I would make size a parameter, move the constant to the place where overview DEFAULT_SIZE is defined and calculate it using Quantity

Reply:
Will be treated in next commits.